bullet_train-super_scaffolding/config/locales/en/scaffolding/completely_concrete/tangible_things.en.yml

```diff
@@ -1,60 +1,35 @@
 en:
-  scaffolding/completely_concrete/tangible_things: &tangible_things
-    label: &label Tangible Things
-    breadcrumbs:
-      label: *label
+  scaffolding/completely_concrete/tangible_things:
+    label: Tangible Things
     navigation:
-      label: *label
       icon: fal fa-puzzle-piece
-    buttons: &buttons
```

Comment:
We can generate a default version of these using `scaffolding/completely_concrete/tangible_things.label` singularized.